openslides/global_settings.py

```diff
@@ -71,7 +67,7 @@
     'django.middleware.locale.LocaleMiddleware',
     'django.middleware.common.CommonMiddleware',
     'django.middleware.csrf.CsrfViewMiddleware',
-    'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'openslides.users.anonymous.AuthenticationMiddleware',
```

Comment:
Can you please rename the authentication submodule to something like `openslides.users.auth`? This file is not only about anonymous users but authentication of all users.